README.md

```diff
@@ -84,6 +84,19 @@ following like to your system `/etc/resolv.conf`:
 nameserver 192.168.8.1
 ```
 
+### Architecture support
+
+Default arch for the test images is `x86_64`
+
+You can export the desired arch and add the matching class with under the `ArchImages` class:
```

Comment:
Please add supported arches, I know right now it's only x86_64 but the follow will have a place to added another supported arches.

Reply:
Changed, is that what you mean?